core/Azure.Mcp.Core/src/Areas/Subscription/Commands/SubscriptionListCommand.cs

```diff
@@ -49,11 +49,9 @@ public override async Task<CommandResponse> ExecuteAsync(CommandContext context,
             var subscriptionService = context.GetService<ISubscriptionService>();
             var subscriptions = await subscriptionService.GetSubscriptions(options.Tenant, options.RetryPolicy);
 
-            context.Response.Results = subscriptions?.Count > 0
-                ? ResponseResult.Create(
+            context.Response.Results = ResponseResult.Create(
```

Comment:
Thanks for catching this! Just a nit change to follow all the other commands doing this:

```dotnet
context.Response.Results = ResponseResult.Create(new(subscriptions ?? []), SubscriptionJsonContext.Default.SubscriptionListCommandResult);
```